dream-hunter/src/components/Result/Result.module.css

```
@@ -0,0 +1,80 @@
+.result {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+
+  margin-block-start: 1rem;
+
+  > li {
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+
+    background-color: var(--color-foreground);
+
+    block-size: 4rem;
+
+    border: 1px solid var(--color-primary-outline);
+    border-radius: 0.5rem;
+
+    box-shadow: var(--shadow);
+
+    .date {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      block-size: 100%;
+      inline-size: 2.75rem;
+
+      background-color: var(--color-primary-outline);
+
+      padding-inline: 0.5rem;
+
+      border-radius: 0.3rem 0 0 0.3rem;
```

حالا جدای از این داستان‌ها، اینجا Parent و Child، دو تا border-raduis مختلف دارن
این باعث شده تو خروجی، رو هم نیفتن و یه ذره از Parent معلوم باشه
اینو با inherit میشه درست کرد
با overflow: hidden هم میشه 

```suggestion
      border-radius: inherit;
      border-start-end-radius: 0;
      border-end-end-radius: 0;
```

---

چون سمت بالا راست و پایین راست می خوام صفر باشه از overflow استفاده کردم